apps/web/src/App.tsx

```diff
@@ -39,6 +39,8 @@ import { Trigger } from './pages/quick-start/steps/Trigger';
 import { RequiredAuth } from './components/layout/RequiredAuth';
 import { TemplateFormProvider } from './pages/templates/components/TemplateFormProvider';
 import { TemplateEditorProvider } from './pages/templates/components/TemplateEditorProvider';
+import { GetStarted } from './pages/quick-start/steps/GetStarted';
+import { DigestPreview } from './pages/quick-start/steps/DigestPreview';
```

Comment:
these files don't exist in this PR, so it's not possible to check how it works :( 
Did you miss it?

Reply:
I committed them by mistake on this PR I added them on https://github.com/novuhq/novu/pull/3006